apps/ehr/src/features/css-module/parser/parser.ts

```diff
@@ -22,7 +22,7 @@ export const getVisitMappedData = (resourceBundle: FhirResource[]): Partial<Visi
 
   return {
     patientName:
-      patientName?.lastFirstMiddleName || patientName?.lastFirstName || patientName?.lastName || patientName?.firstName,
+      patientName?.fullDisplayName || patientName?.lastFirstName || patientName?.lastName || patientName?.firstName,
```

Comment:
could we move this logic into the `getPatientName` function in the `fullDisplayName` itself?
```
const {fullDisplayName } = getPatientName(patient?.name) // fullDisplayName contains patientName?.lastFirstName || patientName?.lastName || patientName?.firstName fallback
```